example-config.yml

```diff
@@ -405,3 +405,7 @@ dateTime:
 #   # departure and it is not entirely excluded from display
 #   # (defaults to 4 days/345600s if unspecified).
 #   timeRange: 345600
+# routeViewer:
+#   # Whether to render routes within flex zones of a route's patterns. If set to true,
+#   # routes will not be rendered within flex zones.
+#   clipRouteRenderToPatternStops: true
```

Comment:
The config parameter name is not very descriptive. Maybe `hideRouteShapesWithinFlexZones`?

Reply:
Good call! Renamed in 04daa4b7000472f9a5b547d37936429ef0824c3b